src/System/src/FloatingBaseSystemDynamics.cpp

```diff
@@ -0,0 +1,182 @@
+/**
+ * @file FloatingBaseSystemDynamics.cpp
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <iDynTree/Model/Model.h>
+#include <iDynTree/Core/Twist.h>
+#include <iDynTree/Core/EigenHelpers.h>
+
+#include <BipedalLocomotion/System/FloatingBaseSystemDynamics.h>
+
+using namespace BipedalLocomotion::System;
+
+FloatingBaseDynamicalSystem::FloatingBaseDynamicalSystem()
+{
+    // set the gravity vector
+    m_gravity.zero();
+    m_gravity(2) = -9.81;
```

Comment:
Here you are assuming a specific orientation for the inertial frame.

Reply:
Yes. However in cbff248063101d95a3f146ae129d015f6c0bc6d0 I introduced the `FloatingBaseDynamicalSystem::setGravityVector()` method that allows defining a custom gravity vector. This goes also in the direction of https://github.com/dic-iit/bipedal-locomotion-framework/pull/46#discussion_r429522585